unstructured_ingest/v2/processes/connectors/astradb.py

```diff
@@ -108,34 +105,38 @@ def precheck(self) -> None:
             raise DestinationConnectionError(f"failed to validate connection: {e}")
 
     @requires_dependencies(["astrapy"], extras="astradb")
-    def get_collection(self) -> "AstraDBCollection":
-        from astrapy.db import AstraDB
+    def get_collection(self) -> Any:
```

Comment:
Why did the typing become less specific?

Reply:
So, to be honest i updated that because thats the typing hint for the return value of the function in AstraPy - i think the issue is it doesn't necessarily return a collection if that collection doesn't exist. Now with that said - if the only choices are Collection or None i agree with not using Any.... but i have to check why in the AstraPy client library its set to Any right now...

Reply:
Ahh, sorry @rbiseck3 - turns out this was an issue in my local development environment. Indeed the type hint for the return is astrapy.Collection. I've pushed a commit to update this accordingly, thanks for calling it out.